src/fiatExchanges/FiatExchangeAmount.test.tsx

```diff
@@ -19,7 +19,14 @@ const mockUseMaxSendAmount = jest.fn(() => mockMaxSendAmount)
 jest.mock('src/fees/hooks', () => ({
   useMaxSendAmount: () => mockUseMaxSendAmount(),
 }))
-jest.mock('src/statsig')
+jest.mock('src/statsig', () => {
+  return {
+    getFeatureGate: jest.fn(),
+    getDynamicConfigParams: jest.fn().mockReturnValue({
+      show_native_tokens: false,
+    }),
```

Comment:
Wonder if there's a way we can avoid including a mock related to an implementation detail of the TokenDisplay component here. Seems like its spread across multiple files, and this is something we'd likely forget to drop if say TokenDisplay no longer depends on this dynamic config any longer

Reply:
yeah, it's not great, though with the refactor to using feature gates again, a lot of these have become redundant/simplified